tests/llvm/test_builtins_matrix.py

```diff
@@ -6,23 +6,27 @@
 
 from llvmlite import ir
 
-DIM_X = 1000
-DIM_Y = 2000
+DIM_X = 100
+DIM_Y = 200
```

Comment:
Is this related to the segfault?
If so it should either adjust the for loop helper to use Phi nodes or wai until #1726 is merged.
Pls, do not hide errors by changing tests.

Reply:
Yes, this is the segfault issue.
Can definitely wait till #1726

Reply:
fixed from changes introduced in #1726